[ADD] surveys: created new 'dashboard' doc

ksc-odoo opened the pull request:
PROJECT TASK: https://www.odoo.com/web#id=3811228&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

- created a new doc to focus SOLELY on the first thing users see when they open the *Surveys* application: the dashboard
- adjusted the main Surveys toctree to add 'dashboard' on the menu

This is part of a continued effort to reorganize/compartmentalize the Surveys doc, and not only have them freshly rewritten and updated for 17 to coincide with the release of the new tutorials, but they will also be broken down in a similar fashion to the tutorials.

Following this, I will write new doc for how to create a Survey -- which will takeover the current content on create.rst (so as not to cause any conflicts with any other doc it may be linked with/to)

Then, I'll write a doc to cover everything about 'Questions' (including Question Types, Question Options, and Conditional Displays)

Followed by other docs for 'Live Sessions' and 'Survey Analysis'

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/8199).

ksc-odoo commented:
This PR is ready for a Peer Review from you -- whenever you get a chance, @Felicious 🙏  Thanks!

Felicious commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,179 @@
+  :guilabel:`Certified`
+
+.. note::
+   If there is a *half-trophy* background image behind the survey name, that means that particular
+   survey is a *Certification*.
```
```suggestion
   A half-trophy background image behind the survey name indicates that the survey is a *Certification*.
```

Felicious commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,179 @@
+      :align: center
+      :alt: The 'Share a Survey' pop-up window that appears in the Odoo Surveys application.
+
+  When the :guilabel:`Send by Email` toggle is in the 'on' position, indicated by a green switch,
```
```suggestion
  When the :guilabel:`Send by Email` toggle is 'on' (green switch),
```

Felicious commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,179 @@
+  When the :guilabel:`Send by Email` toggle is in the 'on' position, indicated by a green switch,
+  additional fields appear, in which :guilabel:`Recipients` and a :guilabel:`Subject` can be added
+  to the email. Below that, a dynamic email template, complete with a :guilabel:`Start
+  Certification` button appears, which can also be modified.
```
```suggestion
  Certification` button is displayed, which can be modified as needed.
```

Felicious commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,179 @@
+  Certification` button appears, which can also be modified.
+
+  Once modifications are complete, click :guilabel:`Send` to send that email invite to all the email
+  addresses/contacts listed in the :guilabel:`Recepients` field.
```
```suggestion
  addresses/contacts listed in the :guilabel:`Recipients` field.
```

Felicious commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,179 @@
+   :alt: The 'Share a Survey' pop-up window in Odoo Surveys with the Send by Email toggled on.
+
+.. important::
+   The :guilabel:`Share` button results in an error if the survey line that it is on has zero
+   questions.
```
```suggestion
   Clicking the :guilabel:`Share` button results in an error if the survey line has zero
   questions.
```

Felicious reviewed (approved):
Amazing job with this doc, @ksc-odoo ! I just had some super minor wording suggestions, and after you take a look at those, feel free to send this along to the next stage of review!

ksc-odoo commented:
Thanks for the feedback, @Felicious -- I made all your suggested edits. This one is now ready for a Tech Review from you, @samueljlieber -- whenever you get a chance. Thanks! 🙏

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+- Percentage and data bar showcasing how many people have :guilabel:`Passed` or become
+  :guilabel:`Certified`
+
+.. note::
+   A half-trophy background image behind the survey name indicates that the survey is a
+   *Certification*.
```
Suggesting to move this note below the bullet points since this is a minor visual cue that is added to Certification surveys, whereas the "Certified" percentage is a stronger visual cue that the record is a Certification survey
```suggestion
```

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+   A half-trophy background image behind the survey name indicates that the survey is a
+   *Certification*.
+
+.. note::
+   The :guilabel:`Passed` percentage and bar **only** appears when a *Required Score* has been
+   configured for that particular survey.
+
+   The :guilabel:`Certfied` percentage and bar **only** appears if that particular survey has the
+   *Is a Certification* option enabled on the survey form.
+
+   If neither :guilabel:`Passed` nor :guilabel:`Certified` appear on the line, that indicates the
+   survey is without a *Required Score* and was not enabled with the *Is a Certification* option.
```
Indent to align with it's relevant bullet point and typo fix
```suggestion
  .. note::
     The :guilabel:`Passed` percentage and bar **only** appears when a *Required Score* has been
     configured for that particular survey.

     The :guilabel:`Certified` percentage and bar **only** appears if that particular survey has the
     *Is a Certification* option enabled on the survey form.

     If neither :guilabel:`Passed` nor :guilabel:`Certified` appear on the line, that indicates the
     survey is without a *Required Score* and was not enabled with the *Is a Certification* option.
```

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+
+- Number of :guilabel:`Courses` related to that survey, which **only** appears if more than one
+  course has been created and attached to a single survey
+
```
Move admonition
```suggestion

.. note::
   A half-trophy background image behind the survey name indicates that the survey is a
   *Certification*.

```

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+
+Those buttons are as follows:
+
+- :guilabel:`Share`: click to reveal a :guilabel:`Share a Survey` pop-up form that can be used to
+  invite potential participants to the survey - complete with a :guilabel:`Survey Link` that can be
+  copied and sent to potential participants, and a :guilabel:`Send by Email` toggle switch.
+
+   .. image:: dashboard/share-survey-popup.png
+      :align: center
+      :alt: The 'Share a Survey' pop-up window that appears in the Odoo Surveys application.
```
This image is not needed since the following displays what this image shows, as well as the advanced configuration.

Also update terminology, this "pop-up" is called a "modal" :)
```suggestion
- :guilabel:`Share`: click to reveal a :guilabel:`Share a Survey` modal that can be used to invite
  potential participants to the survey - complete with a :guilabel:`Survey Link` that can be copied
  and sent to potential participants, and a :guilabel:`Send by Email` toggle switch.
```

ksc-odoo replied:
will delete the image, but am disregarding the "modal" suggestion, per our recent decision re: "modal" vs. "pop-up..."

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+      :align: center
+      :alt: The 'Share a Survey' pop-up window that appears in the Odoo Surveys application.
+
+  When the :guilabel:`Send by Email` toggle is 'on' (green switch), additional fields appear, in
+  which :guilabel:`Recipients` and a :guilabel:`Subject` can be added to the email. Below that, a
+  dynamic email template, complete with a :guilabel:`Start Certification` button appears, which can
+  be modified, as well, if needed.
```
There are a few more fields that appear depending on the configuration of the email
```suggestion
  When the :guilabel:`Send by Email` toggle is active (green switch), additional fields appear, in
  which :guilabel:`Recipients`, :guilabel:`Additional Emails`, and a :guilabel:`Subject` can be
  added to the email.

  Below that, a dynamic email template, complete with a :guilabel:`Start Certification` button
  appears, which can be modified, as well, if needed.

  :guilabel:`Attachments` can be added to the email, as well as an :guilabel:`Answer deadline` can
  be set, if needed.
```

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+   :align: center
+   :alt: The 'Share a Survey' pop-up window in Odoo Surveys with the Send by Email toggled on.
+
+.. important::
+   Clicking the :guilabel:`Share` button results in an error if the survey line has zero questions.
```
It may be helpful to tip where users can edit the default **Mail Template**, as well as note the conditional display of fields depending on config :)
```suggestion
  .. tip::
     The default :guilabel:`Mail Template` for survey invites can be edited by navigating to
     :menuselection:`Settings --> Technical --> Email Templates` and searching for `Survey: Invite`.

  .. note::
     The :guilabel:`Send by Email` toggle switch is **not** present when the survey line has zero
     questions.

     The :guilabel:`Survey Link` only appears when the survey's *Access Mode* is set to *Anyone with
     the link*.

     The :guilabel:`Additional Emails` field only appears when the survey's *Require Login* field is
     **not** active.
```

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+   Clicking the :guilabel:`Share` button results in an error if the survey line has zero questions.
+
+- :guilabel:`Test`: click to take a test version of the survey in a new tab, from the point-of-view
+  of a survey participant, in order to check for any errors or inconsistences.
```
Typo
```suggestion
  of a survey participant, in order to check for any errors or inconsistencies.
```

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+- :guilabel:`See results`: click to reveal a new tab showcasing detailed metrics and graphical
+  representations of all survey participants, questions, and responses for deeper analysis.
+
+- :guilabel:`Start Live Session`: click to initiate a *Live Session* survey, and reveal the *Session
+  Manager* window in a new tab. This button is **not** present for surveys that have enabled the
+  *Is a Certification* option on the survey form.
```
Remove unnecessary formatting and update to casing to "Live session" as it is on the Odoo UI on Survey backend
```suggestion
- :guilabel:`Start Live Session`: click to initiate a *Live session* survey, and reveal a session
  manager window in a new tab. This button is **not** present for surveys that have enabled the
```

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+  Manager* window in a new tab. This button is **not** present for surveys that have enabled the
+  *Is a Certification* option on the survey form.
+
+- :guilabel:`End Live Session`: click to end a *Live Session* survey that has been officially
+  started. This button option **only** appears on survey lines that have previously initiated a
+  *Live Session*.
```
```suggestion
- :guilabel:`End Live Session`: click to end a *Live session* survey that has been officially
  started. This button option **only** appears on survey lines that have previously initiated a live
  session.
```

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+  started. This button option **only** appears on survey lines that have previously initiated a
+  *Live Session*.
+
+Above the buttons that are located to the far-right of the survey lines, there is a mini
+*Configuration* menu that **only** appears when the cursor hovers over that particular line. This
+menu is represented by a :guilabel:`⋮ (three dots)` icon.
+
+When the :guilabel:`⋮ (three dots)` icon is clicked, a drop-down menu with some
+configuration-related options appear.
```
Suggesting to restructure this sentence
```suggestion
Above the buttons that are located to the far-right of the survey lines, a :guilabel:`⋮ (three
dots)` icon appears when the cursor hovers over that particular line. When the :guilabel:`⋮ (three
dots)` icon is clicked, a drop-down menu with some configuration-related options appear:
```

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+When the :guilabel:`⋮ (three dots)` icon is clicked, a drop-down menu with some
+configuration-related options appear.
+
+.. image:: dashboard/three-dot-dropdown.png
+   :align: center
+   :alt: The three-dot drop-down menu of options that appears on the Odoo Surveys dashboard.
+
+The options are:
+
```
Suggesting to move this image below the bullet points and just jump right into the list
```suggestion
```

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+  button at the bottom of the pop-up window.
+- :guilabel:`Color`: users can opt to choose a color to add to the survey line on the dashboard for
+  added organizational purposes, if needed.
+
```
```suggestion

.. image:: dashboard/three-dot-dropdown.png
   :align: center
   :alt: The three-dot drop-down menu of options that appears on the Odoo Surveys dashboard.

```

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,178 @@
+Doing so reveals a table with rows and columns. The rows show the different surveys in the database,
+and the columns depict the various activity types.
+
+.. important::
+   The :guilabel:`New` button does **not** appear in this view.
```
I don't feel that this is _important_ information, rather it is just a note. It may be helpful to note the use-case of the view rather than just what _cannot_ be done while on this view
```suggestion
.. note::
   A new survey cannot be created in this view, as it is solely for the purpose of creating and
   viewing scheduled activities.
```

samueljlieber commented on content/applications/marketing/surveys/dashboard/activities-view.png:
The text in this image is tough to read, and may be able to be retaken with more clarity and less width?

samueljlieber commented on content/applications/marketing/surveys/dashboard/share-survey-popup.png:
Suggesting to remove this image

samueljlieber reviewed (changes requested):
Hi @ksc-odoo! Awesome job on this brand new Surveys doc, it is coming along nicely! I have some suggestions and corrections for you, please see below. 

I'd also like to suggest using either this `dashboard.rst` content or `create.rst` content as the main `surveys.rst` content, unless you plan on writing overview content specifically for `surveys.rst`. This would help remove an "extra step" towards getting to content in the TOC, let me know your thoughts!

Once these are addressed, tag me again for another quick look. Looking forward to seeing more Surveys content! :) Thank you!

ksc-odoo commented:
@samueljlieber Thanks for all the great feedback -- i really appreciate it 🙏 Just made all your suggested edits. Ready for another look, whenever you get a chance. Thanks!

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,188 @@
+
+- :guilabel:`See results`: click to reveal a new tab showcasing detailed metrics and graphical
+  representations of all survey participants, questions, and responses for deeper analysis.
+
```
```suggestion
```

samueljlieber commented on content/applications/marketing/surveys/dashboard.rst:
```diff
@@ -0,0 +1,188 @@
+- :guilabel:`Start Live Session`: click to initiate a *Live session* survey, and reveal a session
+  manager window in a new tab. This button is **not** present for surveys that have enabled the *Is
+  a Certification* option on the survey form.
+
```
```suggestion
```

StraubCreative reviewed (changes requested):
Hi @ksc-odoo 

At a quick glance I'm thinking this type of doc would serve better as body content for `surveys.rst` since we're covering UI and settings. Those directory parent files are more ideal for this, whereas the children articles underneath them serve better as specific flows— anything that produces a desired outcome.

Can you make the adjustment, please, and then I'll take another look at the content? Thanks!

ksc-odoo commented:
Alrighty, @StraubCreative -- I moved the dashboard content to the body of the Surveys.rst file, per your request -- and made sure to move all the images, as well *and* deleted the original dashboard.rst file, so I believe this is ready for another look from you whenever you get a chance. Thanks! 👍
